docs/2.guide/2.directory-structure/1.content.md

```diff
@@ -41,7 +41,7 @@ To render content pages, add a [catch-all route](/docs/guide/directory-structure
 ```vue [pages/[...slug\\].vue]
 <template>
   <main>
-    <ContentDoc :path="$route.path" />
+    <ContentDoc :path="$route.params.slug" />
```

Comment:
We might actually consider removing the `:path="$route.path"` entirely, as this is the default value when no `path` prop is passed.

https://content.nuxt.com/components/content-doc#default